tests/gradients/test_finite_difference.py

```diff
@@ -180,27 +180,46 @@ def test_independent_parameter(self, mocker):
         # only called for parameter 0
         assert spy.call_args[0][0:2] == (tape, 0)
 
-    def test_no_trainable_parameters(self, mocker):
-        """Test that if the tape has no trainable parameters, no
-        subroutines are called and the returned Jacobian is empty"""
-        spy = mocker.spy(qml.gradients.finite_difference, "generate_shifted_tapes")
+    @pytest.mark.parametrize("interface", ["autograd", "jax", "torch", "tf"])
+    def test_no_trainable_params_qnode(self, interface):
+        """Test that the correct ouput and warning is generated in the absence of any trainable
+        parameters"""

```

Comment:
Thank you for adding these parameterizations! :)

@dime10 only one CI action installs the non-standard interfaces (all interfaces except for Autograd). Therefore some of the checks will likely error as `jax`, `torch` or `tensorflow` is not installed.

Skipping the test can be a good way forward, if the import is not successful. Note: the suggestion was not tested locally.
```suggestion
        parameters"""
        interface_name = interface if interface != "tf" else "tensorflow"
        pytest.importorskip(interface_name)
```

Reply:
Ah good point, I will change that!